keps/0000-kep-template.md

```diff
@@ -40,11 +40,8 @@ To get started with this template:
 1. **Pick a hosting SIG.**
   Make sure that the problem space is something the SIG is interested in taking up.
   KEPs should not be checked in without a sponsoring SIG.
-1. **Allocate a KEP number.**
```

Comment:
Do we also need to update the above paragraph?

> The *filename* for the KEP should include the KEP number along with the title.
The title should be lowercased and spaces/punctuation should be replaced with `-`.
As the KEP is approved and an official KEP number is allocated, the file should be renamed.

Reply:
Do we also need to remove `kep-number: 0` at line 2?